jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ProxyConnectionFactory.java

```diff
@@ -204,25 +193,36 @@ public void onFillable()
             }
 
             @Override
-            public void onOpen()
+            public void onFillable()
```

Comment:
The fact that you flipped `onOpen()` and `onFillable()`'s order makes this review unnecessarily more complex to follow, as it makes it more complex to find back the places where you added the missing `release()` calls.

Could you please restore their original order?